classes/install/form/InstallForm.php

```diff
@@ -44,7 +44,8 @@ class InstallForm extends MaintenanceForm
         // <driver> => array(<php-module>, <name>)
         'mysqli' => ['mysqli', 'MySQLi'],
         'postgres9' => ['pgsql', 'PostgreSQL'],
-        'mysql' => ['mysql', 'MySQL']
+        'mysql' => ['mysql', 'MySQL'],
+        'mariadb' => ['mysql', 'MariaDB'],
```

Comment:
Should this be `mysql` inside the array, or `mysqli`?

Reply:
> Should this be mysql inside the array, or mysqli

should be `mysqli` .